spark/src/main/resources/interpreter-setting.json

```diff
@@ -77,6 +77,12 @@
         "propertyName": "zeppelin.spark.maxResult",
         "defaultValue": "1000",
         "description": "Max number of SparkSQL result to display."
+      },
+      "zeppelin.spark.importImplicit": {
+        "envName": "ZEPPELIN_SPARK_IMPORTIMPLICIT",
+        "propertyName": "zeppelin.spark.importImplicit",
```

Comment:
Though I would argue that you can still use this option, the whole idea of it is when set to `true`, user is supposed to import all implicits, I would actually have only one option for this, not many.